.buildkite/pipeline.elastic-agent-binary-dra.yml

```diff
@@ -101,6 +144,22 @@ steps:
         DRA_WORKFLOW: "dra-core-staging"
         PLATFORMS: "linux/arm64 darwin/arm64"
 
+    - label: ":package: linux/arm64 darwin/arm64 Elastic-Agent Core staging"
```

Comment:
I do not see any references to FIPS in the label. Is that intentional? Is FIPS applicable only to `linux/arm64` or also to `darwin/arm64`?

Reply:
Forgot to change the label. Good catch!
Fixed in d738701672aaf5c8c211cd67aa314c356b2e676e